src/templates/Pool/NewPoolOperations/Form/Invest/index.tsx

```diff
@@ -399,6 +404,12 @@ const Invest = ({ typeAction, privateInvestors }: IInvestProps) => {
       Big(slippageVal.replace('.', ''))
     )
 
+    let referrerAddress = ZeroAddress
+    if (referralData) {
+      const checkWalletAddress = isAddress(referralData.value)
+      referrerAddress = checkWalletAddress ? referralData.value : ZeroAddress
```

Comment:
Idk if you guys have a pattern for boolean values, but I guess you should. `check` is not a well named variable for boolean in my opinion, it could be something like `isWalledAddress` or `hasWalledAddress`.

```suggestion
      const isWalletAddress = isAddress(referralData.value)
      referrerAddress = checkWalletAddress ? referralData.value : ZeroAddress
```

OR, since it's not being used anywhere lese in this `if`, you could use it directly and save your creativity for naming variables haha(probably better this way)

```suggestion 
      referrerAddress = isAddress(referralData.value) ? referralData.value : ZeroAddress
```